src/components/Player/Opsheet/ListType.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+
+class ListType extends Component {
+    render() {
+      return (
+        <div className='list-types-selection-wrapper'>
+            None<input type="checkbox" value='none' name="" id=""/>
+            Real Icon<input type="checkbox" value='real' id=""/>
+            Fakes Icon<input type="checkbox" value='fakes' id=""/>
+        </div>
+      );
+    }
+  }
+ 
+export default ListType;
```

Comment:
Try not to get frustrated if you notice the same "poor form" called out repeatedly in PRs. It might feel like someone is harping on you, but in practice, it's convenient to have someone else place markers in your code to make it easy to locate things you intend to fix.